services/QuillLMS/app/controllers/cms/districts_controller.rb

```diff
@@ -0,0 +1,253 @@
+# frozen_string_literal: true
+
+class Cms::DistrictsController < Cms::CmsController
+  before_action :signed_in!
+
+  before_action :text_search_inputs, only: [:index, :search]
+  before_action :set_district, only: [
+    :show
+  ]
+
+  DISTRICTS_PER_PAGE = 30.0
```

Comment:
This should be an integer.
```suggestion
  DISTRICTS_PER_PAGE = 30
```